src/editor/diagnostics.ts

```diff
@@ -7,13 +7,19 @@ import {
     Location,
     Position,
     Range,
+    ThemeColor,
     Uri,
     languages,
     window
 } from 'vscode';
 import { ExtensionApi } from '../backend/api';
 import { DiagnosticReport } from '../backend/types';
 
+// Decoration type for highlighting report step positions.
+const reportStepDecorationType = window.createTextEditorDecorationType({
+    backgroundColor: new ThemeColor('editorMarkerNavigationWarning.background')
```

Comment:
Really not sure about this color choice...
![image](https://user-images.githubusercontent.com/16914176/154246054-7783abe3-383d-49fb-935e-49eee6894dfb.png)
Played around with a couple other themes, about 50% of them look like this, both light and dark.
I think it would be better to have our own contributed color here.

Reply:
@Discookie I added new configurable colors and updated the screenshots in this PR message above.

Reply:
Yep, that works now for most themes, thanks!